.github/workflows/linux.yml

```diff
@@ -132,9 +132,17 @@ jobs:
 
     - name: Measure coverage
       if: ${{ matrix.build-type == 'Coverage' }}
-      timeout-minutes: 5
+      timeout-minutes: 10
+      shell: /bin/bash -xeuo pipefail {0}
       run: |
         sg lxd -c '/snap/bin/lxc start snapcraft-multipass'
+        timeout 10 sg lxd -c \
+          'while [ "$( /snap/bin/lxc exec snapcraft-multipass -- \
+             systemctl show --property=ActiveState snapd )" != "ActiveState=active" ]; do sleep 1; done'
+        sg lxd -c \
+          '/snap/bin/lxc exec snapcraft-multipass -- \
+             sh -c "sudo sed -i \"s/use JSON/use JSON::XS/\" \`which geninfo\`"'
+        sg lxd -c '/snap/bin/lxc exec snapcraft-multipass -- apt-get -y install libjson-xs-perl'
```

Comment:
Add a comment why all this?

Reply:
Will do.

Reply:
Ok, done and rebased into a previous related commit.